ports/qtbase/portfile.cmake

```
@@ -79,6 +79,10 @@ INVERTED_FEATURES
 list(APPEND FEATURE_OPTIONS -DCMAKE_DISABLE_FIND_PACKAGE_Libudev:BOOL=ON)
 list(APPEND FEATURE_OPTIONS -DFEATURE_xml:BOOL=ON)
 
+if(NOT QT_NAMESPACE STREQUAL "")
```

This does not correctly handle `QT_NAMESPACE` being undefined. For example,
```
$ cat cmaketest.cmake
if(NOT ABC STREQUAL "")
    message("bang")
endif()
$ cmake -P cmaketest.cmake
bang
```
This needs to be

```suggestion
if(QT_NAMESPACE)
```